Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocSimpleColumnDefinitionTest.cs

```diff
@@ -66,5 +69,46 @@ public void GetValidationFailureMatcher_ReturnsInstanceOfPropertyPathValidationF
           ((BusinessObjectPropertyPathValidationFailureMatcher)validationFailureMatcher).PropertyPath,
           Is.EqualTo(column.GetPropertyPath()));
     }
+
+    [Test]
+    public void ColumnTitleDisplayValue_EmptyColumnTitle_StaticPropertyPath_ReturnsDisplayNameOfLastProperty ()
```

Comment:
Do we also want a test for this case with a last property = null?

Reply:
Yes, good idea. I added that.